### PR TITLE
Write first draft of June entry

### PR DESCRIPTION
**Pull request by janhohenheim**
[Rendered](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/june-draft/content/news/052/index.md)

This is the first draft of the June edition of the newsletter. Feel free to merge already, there will be another PR for the actual release.

_Part of #1523_

### CODE REVIEW

File: content/news/052/index.md [hunk 1]
```diff
@@ -70,15 +67,14 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Game Updates
 
 ### [Untitled Pixel Wizards Game][pixel-wizards]
 
 {{ embed_video(type="video/mp4", src="untitled-pixel-wizards-game.mp4",
-caption="Enemies now perceive, pursue and attack.. and occasionally get burned to death.") }}
-
-[![A Hound chases a player, and its corpse ragdolls after it is burned to death by a fireball](untitled-pixel-wizards-game.gif)][pixel-wizards]  
-_Enemies now perceive, pursue and attack.. and occasionally get burned to death._
+caption="Enemies now perceive, pursue and attack.. and occasionally get burned to death.") }}
```
Comment (Vrixyz) [inline content/news/052/index.md:77]: What’s this caption ? It seems great ; is it visible ?

File: content/news/052/ssr.png
Comment (Vrixyz) [inline content/news/052/ssr.png:1]: Was this image optimized in size ?

File: templates/shortcodes/embed_video.html [hunk 2]
```diff
@@ -5,7 +5,5 @@
 
 #} -->
 <video controls autoplay muted loop>
-    <source
-        type="{{ type }}" src="{{ src }}"
-        onerror="parentNode.parentElement.innerText = 'Sorry, this video does not exist or your browser does not support the video codec. Try a different browser!'" />
-</video> {% if caption %}<em>{{ caption }}</em>{% endif %}
+    <source type="{{ type }}" src="{{ get_url(path=page.colocated_path ~ src)}}" />
+</video> {% if caption %}<em>{{ caption }}</em>{% endif %}
```
Comment (Vrixyz) [inline templates/shortcodes/embed_video.html:9]: Ah that’s the caption ?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -46,6 +46,7 @@ jobs:
           --exclude '[^\w]todo[^\w]?'
           --exclude 'https://(www\.|old\.)?reddit\.com'
           --exclude 'https://www.patreon.com'
+          --exclude 'https://gamedev.social'
```
Comment (Vrixyz) [inline .github/workflows/ci.yml:49]: Probably not to keep ultimately

Comment (janhohenheim) [reply .github/workflows/ci.yml:49]: Yep, I'll remove it again once they get their stuff together 😅

File: content/news/052/index.md (hunk 1 above)
Comment (janhohenheim) [reply content/news/052/index.md:77]: image
Yep, looks like this :)

File: templates/shortcodes/embed_video.html (hunk 2 above)
Comment (janhohenheim) [reply templates/shortcodes/embed_video.html:9]: Exactly!

File: content/news/052/ssr.png
Comment (janhohenheim) [reply content/news/052/ssr.png:1]: I did not optimize any media yet

File: content/news/052/index.md (hunk 1 above)
Comment (Vrixyz) [reply content/news/052/index.md:75]: looking great ! I found more info here: https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1529

File: content/news/052/ssr.png
Comment (janhohenheim) [reply content/news/052/ssr.png:1]: Update: done, ran them all through https://squoosh.app

**Review by mamaicode — APPROVED** [review]

**Pull request merged**